global.js

```diff
@@ -0,0 +1,9 @@
+require('node-libs-react-native/globals')
+
+global.navigator.userAgent = ''
+const navigatorCopy = { ...global.navigator }
```

Comment:
i think the issue has been fixed in isomorphic-webcrypto, check if they have a new version on npm